content/engineering/our-approach/release-checklist.md

```diff
@@ -0,0 +1,102 @@
+---
+title: Release checklist
+sidenav: true
+sticky_sidenav: true
+tags: engineering
+permalink: /engineering/our-approach/release-checklist/
+redirect_from:
+  - engineering/release-checklist/
+layout: layouts/page
+eleventyNavigation:
+  parent: engineering_approach
+  key: Release checklist
+  order: 10
+  title: Release checklist
+subnav:
+  - text: Introducing release checklist
+    href: "#introducing-release-checklist"
+  - text: What should it look like?
+    href: "#what-should-it-look-like"
+  - text: Questions to consider
+    href: "#questions-to-consider"
+  - text: Resources
+    href: "#resources"
+---
+## Introducing release checklist
+
+The release checklist is a list of all tasks that need to be completed during the release process -- be it a launch or updating software. The checklist helps ensure that all tasks are completed prior to a release, reducing risk, and helps with following up on what needs to be done once release is complete.  When people see all the tasks listed and who owns each item, it increases confidence that the release will be successful. 
+
+Being transparent and communicating about what is going on is very important to the success of the release. We encourage people to follow and document everything on the checklist so it can be revisited after release and use that information to improve the checklist for the upcoming release. 
+
+## What should it look like?
+
+The following "columns" are recommended:
+
+>**Owner**: Who owns this item in the checklist?
+
+>**Date and Time**: What is expected for this to be completed? How long will it take to complete this?
+
+>**Explanation of Task**: What is expected of the task being completed in this?
+
+>**Status**: What is the status of this? It can be set to the following fields: Pending, Started/In Progress, Completed/Done, Blocked
+
+>**Notes**: Any notes that would like to be reviewed at a later time. This can be used for reflection meetings and to improve the checklist after the release for the next release.
```

Comment:
Feel free to reject. As I was reading it, though, I was a little thrown that these bullet points were in a different order from the example table columns. 😛 

```suggestion
>**Date and Time**: What is expected for this to be completed? How long will it take to complete this?

>**Owner**: Who owns this item in the checklist?

>**Status**: What is the status of this? It can be set to the following fields: Pending, Started/In Progress, Completed/Done, Blocked

>**Explanation of Task**: What is expected of the task being completed in this?

>**Notes**: Any notes that would like to be reviewed at a later time. This can be used for reflection meetings and to improve the checklist after the release for the next release.
```